src/App.tsx

```diff
@@ -1,10 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [keyPressed, setKeyPressed] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const detectKeyDown = (e: KeyboardEvent) => {
+    setKeyPressed(e.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keyup', detectKeyDown);
+
+    return () => document.removeEventListener('keyup', detectKeyDown);
+  }, []);
+
+  return (
+    <div className="App">
+      {/* Keep this input for tests */}
+      <input type="text" />
+      {
+        keyPressed
```

Comment:
```suggestion
      {keyPressed
```

Reply:
@naziamoff @meforoma @SalehBUD @Rijo-J 
 Thank you everyone for your time and patience!